docs/supported_ops.rst

```diff
@@ -1,44 +1,54 @@
+- **Sequences** means that the operator can produce or accept as an input a sequence, or video type of input.
 - **Volumetric** means that the operator supports 3D data processing.
 
-How to read this doc
+Reading this guide
 ^^^^^^^^^^^^^^^^^^^^
 
-DALI Operators are used in two steps - creating the parametrized Operator instance using
-its constructor and later invoking its `__call__` operator
-in :meth:`~nvidia.dali.pipeline.Pipeline.define_graph` method of the Pipeline.
+DALI operators are used to create the parametrized operator instance by using
+its constructor and later to invoke its `__call__` operator
+in the pipeline’s :meth:`~nvidia.dali.pipeline.Pipeline.define_graph` method.
 
-Documentation of every DALI Operator lists **Keyword Arguments** supported by the class constructor.
+In the documentation of for every DALI Operator, see the lists of **Keyword Arguments**
+that are supported by the class constructor.
 
-The documentation for `__call__` operator lists the positional arguments (**Parameters**) and additional **Keyword Arguments**.
-`__call__` should be only used in the :meth:`~nvidia.dali.pipeline.Pipeline.define_graph`.
-The inputs to the `__call__` operator represent batches of Tensors (TensorLists) processed by DALI,
-which are returned by other DALI Operators.
+The documentation for ``__call__`` operator lists the positional arguments, (or parameters) and additional
+keyword arguments. ``__call__`` should only be used in the :meth:`~nvidia.dali.pipeline.Pipeline.define_graph`.
+The inputs to the ``__call__`` operator represent batches of Tensors (TensorLists) that are processed by DALI and
+are returned by other DALI operators.
 
-The **Keyword Arguments** listed in `__call__` operator accept TensorList argument inputs. They should be
-produced by other 'cpu' Operators.
+The **keyword arguments** in ``__call__`` operator accept TensorList argument inputs and should be
+produced by other cpu operators.
 
 .. note::
-    The names of positional arguments for `__call__` operator (**Parameters**) are only for the
+    The names of the positional arguments for ``__call__`` operator (**parameters**) are provided only for
     documentation purposes and should not be used as keyword arguments.
 
 .. note::
-    Some **Keyword Arguments** can be listed twice - once for class constructor and once for `__call__` operator.
-    This means they can be parametrized during operator construction with some Python values
-    or driven by output of other operator when running the pipeline.
+    Some keyword arguments can be listed twice. Once for the class constructor and once for
+    ``__call__`` operator. This listing means the arguments can be parametrized during operator
+    construction with some Python values or driven by the output from another operator when
+    running the pipeline.
 
 
-Support table
+Support Table
 ^^^^^^^^^^^^^
 
+The following table lists all available operators and devices on which they can operate:
```

Comment:
```suggestion
The following table lists all available operators and devices on which they can run:
```
The operators operate on data, run on device.

Reply:
Done